packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -41,6 +41,7 @@ const generateComponentMeta = (): void => {
   requiredRootNode?: TagName[]; // components, that use this internal component within their shadow DOM
   requiredChild?: string; // direct and only child of kind
   requiredChildSelector?: string; // might contain multiple selectors separated by comma
+  requiredSlots?: { slot: string; slotTagName: TagName }[];
```

Comment:
Could be confusing. I guess `slot` would be `slotName` while the other one is the `slottedTagName`?
I'd suggest to use the correct terminology, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot and https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots